recipes/eleuther_eval.py

```diff
@@ -33,6 +33,16 @@
     sys.exit(1)
 
 
+def _parent_name(target):
```

Comment:
Where is this used? I didn't find this in the file below

Reply:
oh will remove, this was used for workaround issues in tensor subclass